project.clj

```diff
@@ -50,4 +51,5 @@
           :defaults {:doc/format :markdown}}
   :source-paths ["src/clj"]
   :java-source-paths ["src/java"]
+  :resource-paths ["src/resources"]
```

Comment:
Include the `src/resources` directory which now includes an implementation of the `HttpFieldPreEncoder` 'service' from the `http2-hpack` code.